src/interchange/src/avro/envelope_cdc_v2.rs

```diff
@@ -46,7 +46,7 @@ pub fn extract_data_columns<'a>(schema: &'a Schema) -> anyhow::Result<SchemaNode
 #[derive(Debug)]
 pub struct Encoder {
     columns: Vec<(ColumnName, ColumnType)>,
-    schema: Schema,
+    _schema: Schema,
```

Comment:
This one's dead code too, and looks like it can just be deleted. @umanwizard do you agree?

Reply:
I don't think we use this `Encoder` struct at all, except in tests. @guswynn can you check if that's true and move it into the test module if so?